efd/salkafka/Dockerfile

```diff
@@ -1,10 +1,13 @@
-FROM lsstts/deploy-env:salobj_v5.4.0_idl_v1.1.2_xml_v4.7.0
+FROM lsstts/deploy-env:salobj_v5.7.0
 
```

Comment:
Since salkafka has its own repo, we should move this Dockerfile there. Since it won't be done now, can a ticket be filed to move it?

Reply:
This can probably be done in bulk so I'll make a generic ticket (DM-24363).